src/DssSpellCollateralOnboarding.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.12;
+
+import "dss-exec-lib/DssExecLib.sol";
+
+contract DssSpellCollateralOnboardingAction {
+
+    // --- Rates ---
```

Comment:
I would include the following comment before rates one:
```
    // Many of the settings that change weekly rely on the rate accumulator
    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
    // To check this yourself, use the following rate calculation (example 8%):
    //
    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
    //
    // A table of rates can be found at
    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
    //
```

Reply:
Also, we should consider updating the IPFS table of rates to include the missing 0.1% rate that was used in previous spell.

Reply:
> Also, we should consider updating the IPFS table of rates to include the missing 0.1% rate that was used in previous spell #208

I think that is a good idea, but that doesn't feel like it fits in here?  How do we update that?

Reply:
We could open a separate PR for this. 
I think we can update it via Pinata, by pinning an updated table of rates to IPFS.

Reply:
To consider in #212

Reply:
Just pinned an updated table of rates (including 0.01% and 0.10%, based on `rates.sol`) via pinata:
- [QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj](https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj)
- [diffcheck](https://www.diffchecker.com/CfMqmMKp)

Reply:
Will probably need to backport this to spells-goerli also.

Reply:
added to mainnet here: https://github.com/makerdao/spells-mainnet/pull/212/commits/722a5cb4973135014cef86e1abae1d2e88c84cda